addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6664,7 +6679,14 @@ registry.BackgroundPosition = SnippetOptionWidget.extend({
             this._toggleBgOverlay(false);
         });
 
-        this.$backgroundOverlay.insertAfter(this.$overlay);
+        if (this.$overlay[0].ownerDocument !== this.$target[0].ownerDocument) {
+            this.iframeManipulators = document.createElement('div');
+            this.iframeManipulators.id = 'oe_manipulators';
+            this.targetWindow.document.body.appendChild(this.iframeManipulators);
+            this.$backgroundOverlay.appendTo(this.iframeManipulators);
+        } else {
+            this.$backgroundOverlay.insertAfter(this.$overlay);
+        }
```

Comment:
Meh ^^ Sorry you lost you kinda lost your time: I have been working thursday and friday on moving back the scroll on the html instead of the wrapwrap and as part of that work (unfinished), I will move back all overlays inside the iframe, as explained here: https://github.com/odoo/odoo/pull/98429/commits/c253c69cddb942b6a385da508ea8bb27af884cc4 I also adapted the background option in that PR. I also checked with PDE today who agreed that the background sizing can be simplified by removing the apply/discard buttons so I am removing them.

I'll check if something of your PR should remain. I guess it should as my only change was this I think:
![image](https://user-images.githubusercontent.com/10338094/185923004-fac1dd58-eda8-4c95-ab84-2f9613374515.png)
Again, sorry for the overlap. Move on to something else, independent of the overlay/scroll for now.

Reply:
Did you have the chance to merge your imp about it @qsm-odoo ?

Reply:
I don't have any chance

Reply:
What do you mean, you are working with me, if that ain't being lucky, what is it?

Reply:
@detrouxdev I merged my commit which moved the whole overlay back inside the website content (iframe). I don't think any of this PR is relevant anymore... but I could be wrong. Could you have a look and if any of it is necessary, could you describe the exact flow that are broken in the commit message? (Otherwise, close the PR and delete the branch of course). Thank you :+1: 

Note, I was not in time to merge the two related imp I wanted, I will do that for 16.1:
- Scrollbar back on html instead of wrapwrap
- No Save/discard buttons for background sizing UI (just auto save)